support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -119,4 +120,28 @@ export const tests: Tests = {
 		targetPage: '/subscribe$',
 		excludeContributionsOnlyCountries: true,
 	},
+	newspaperGenericCheckout: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'variant',
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 0,
+			},
+		},
+		isActive: true,
+		referrerControlled: false, // ab-test name not needed to be in paramURL
+		seed: 9,
+		targetPage: pageUrlRegexes.subscriptions.paper.paperLandingPage,
+		persistPage:
+			// match generic checkout & thank you page
+			'^/uk/(checkout|thank-you)',
```

Comment:
ah yes, you will be in either ab-test through these pages too. Don't think I need to do anything extra on the thank-you page.

Reply:
Will users in the `control` be bucketed in the `newspaperGenericCheckout` test when they proceed to the existing checkout path: `https://support.theguardian.com/subscribe/paper/checkout`?